### ISSUE TITLE
Fix storybook issues related to Vite migration

### ISSUE THREAD

**Issue by blushi** [regen-network/regen-web#1974]
As part of the [migration to Vite](https://github.com/regen-network/regen-web/pull/1955), we decided to merge as is and address a few Storybook related issues as part of this follow-up issue:

- It seems like in some cases, the storybook local dev server gets restarted: https://github.com/storybookjs/storybook/issues/22808 (it seems to be working with a fresh new install in the mean time), probably we should wait on some updates upstream
- We've just ignored the stories in web-registry because they were causing some issues with the storybook build. We should figure this out and add them back or have a different setup with different storybook instances running for the different packages
- There was an issue with `Buffer` being undefined in `modal.stories.tsx`, we should fix the storybook vite config to polyfill/include that (ref: https://github.com/regen-network/regen-web/pull/1955#issuecomment-1641744301)

**Comment by ryanchristo** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/RND-Dev-Team-5f8998bec8958d000f4609e2/issues/regen-network/regen-web/1974?planning-poker) with Zenhub @flagrede @ryanchristo @wgwz

**Comment by flagrede** [comment]
Some updates on this task:
Our issue comes from the `fsevents` library, more details on that here: https://github.com/regen-network/regen-web/pull/2038#discussion_r1304334652

I tried to:
- update the dependency to `2.2.3` as this contains the fix but this break the CI
- patch it with `patch-package` but we still had the issue with storybook locally.
- patch it with yarn v3, which solve the issue with storybook but I had issues with `web-www` due to the yarn upgrade (had also to replace lerna with yarn workspace). My progress on that path is available on this branch `fix-1974-fix-storybook-yarn-worspace`
